deep_quoridor/test/quoridor_env_test.py

```diff
@@ -68,16 +68,16 @@ def parse_board(board):
 
                 if ch == "v" or ch == "|":
                     forbidden_walls.append((row_n, col_n - 1, 0))
-                    if ch == "|" and not env.is_wall_between(row_n, col_n - 1, row_n, col_n):
-                        env.place_wall("player_0", (row_n, col_n - 1), 0)
+                    if ch == "|":
+                        game.step(WallAction((row_n, col_n - 1), WallOrientation.VERTICAL))
```

Comment:
I'm not sure if this would work as intendeed, because this was a little hack so that the wall is only placed on the top/left part.  If you don't check, you may place another wall that will extend it. E.g., when we find --, we need to place only one wall starting on the left, if you don't check if there's a wall already, you'll place another wall and will be ---